gourmet/gtk_extras/pageable_store.py

```diff
@@ -167,17 +167,13 @@ def sort (self, column, direction=FORWARD):
 
         Note -- to remove term we use direction=OFF
         """
+        assert(direction in (self.FORWARD, self.REVERSE, self.OFF))
 
         self.sort_dict[column]=direction
         if direction==self.OFF:
             self.parent_list = self.unsorted_parent
             return
-        if direction==self.FORWARD: shift_by = 1
-        elif direction==self.REVERSE: shift_by = -1
-        self.parent_list.sort(lambda r1,r2: ((r1[column]>r2[column] and 1*shift_by) or
-                                             (r1[column]<r2[column] and -1*shift_by) or
-                                             0)
-                              )
+        self.parent_list.sort(key=lambda x: x[column], reverse=(direction == self.REVERSE))
```

Comment:
I think the reversal condition is wrong, but the original looks wrong in that case